CMakeLists.txt

```diff
@@ -4,13 +4,18 @@ project(DoubleDispatch VERSION 1.0 LANGUAGES CXX)
 
 set(CMAKE_CXX_STANDARD 11)
 
-# Manage dependencies
+if (UNIX)
```

Comment:
@Morheit , why you need this IF? for me it looks like redundant

Reply:
Removed